### ISSUE TITLE
keyboard not handled 100% correct

### ISSUE THREAD

**Issue by danielehrhardt** [codext-remotecontrol/remotecontrol-desktop#5]
When using special characters, it can happen that they are not transferred.

**Comment by test2a** [comment]
https://github.com/codext-remotecontrol/remotecontrol-desktop/assets/39449563/e4430179-6da3-4ae4-900d-52588703d66c


not just special characters, pressing capslock button actually types CAPSl0CK every time.

secondly, when caps is off, shift + letter does not make it capital, it is always small.

when caps is on, shift + letter makes it small otherwise it is caps

**Comment by test2a** [comment]
also, when we have the remote window open, we expect the software to capture keybindings like alt-tab. it should happen on remote desktop and not local.

**Comment by danielehrhardt** [comment]
> also, when we have the remote window open, we expect the software to capture keybindings like alt-tab. it should happen on remote desktop and not local.

yes this is true. Currently, i have to check how much time i have to check this again, it is tricky to make this work with all languages. If you find someone that can contribute, that would help a lot.

**Comment by test2a** [comment]
also, i am unable to type " or : or any character while holding shift key so no @ or ! or any of such characters
